### PR TITLE
LPS-120518 - Fix outline secondary button color

### PR DESCRIPTION
**Pull request by marcoscv-work**
This PR fixes an outline secondary button color introduced by custom properties/tokens.

---

To test it:

1º Create a new content page with an HTML fragment with this code:
`<button class="btn btn-outline-secondary">btn-outline-secondary</button>`

2º Add a new stylebook and check the button outline secondary text color

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
ci:test:relevant

File: modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json [hunk 1]
```diff
@@ -522,7 +522,7 @@
 							"type": "String"
 						},
 						{
-							"defaultValue": "#272833",
+							"defaultValue": "#6b6c7e",
```
Comment (jbalsas) [inline modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json:525]: We have this same colour in other places as `#6B6C7E`. Unless I'm mistaken, we're using upper case for HEX colours.

File: modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss [hunk 2]
```diff
@@ -10,7 +10,7 @@ $lead-font-weight: 400;
 $small-font-size: 0.875rem;
 
 $btn-outline-secondary: (
-	color: $dark-color,
+	color: #6b6c7e,
```
Comment (jbalsas) [inline modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss:13]: I thought we were always using upper case for our HEX values... it doesn't look like that's the case in here which I'm not sure is a new issue or an old one...

File: modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json (hunk 1 above)
Comment (jbalsas) [reply modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json:525]: There's actually a big mix of upper and lower case here... 😭 

This can become an issue when "grepping" for occurrences with case sensitive settings that we might fail to find the same colour because it uses different combination of upper and lower case letters.

@wincent, I thought we had a rule for this? This is of course a `json` file, so it won't apply here, but wondering if the rule is all lowercase now as seen in the `.scss` in this same PR.

In either case, we should have a single format here, I guess

Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json:525]: oh! that's true!

I'm going to fix this and the file!

File: modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss (hunk 2 above)
Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss:13]: It looks like we have been using lower case for a long time and in other components like color picker are working on lowercase too so we can see in first place `#FFFFFF` but then if you modify it changes to `#ffffff`.
I going to move all uppercases to lower cases in `frontend-token-definition.json` :-D

File: modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json (hunk 1 above)
Comment (wincent) [reply modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json:525]: > @wincent, I thought we had a rule for this? This is of course a json file, so it won't apply here, but wondering if the rule is all lowercase now as seen in the .scss in this same PR.

You basically said it:

- We don't enforce anything like this inside JSON.
- "We" do enforce it in SCSS, by which I mean, Prettier enforces it (to lowercase).

I think we could pretty easily do this in JSON using liferay-npm-scripts, at least for the obvious case; ie:

- Strings exactly matching `/^[a-f0-9]{3,6}$/i`.

I mean, you might end up formatting something you don't meant to format (eg. a hashtag ... 🤦‍♂️) but we could always ignore such edge case if it ever came up.

Created https://github.com/liferay/liferay-npm-tools/issues/494 to track this.

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by pat270** [comment]
LGTM! The rule in the past was uppercase hex values and no hex shorthand.

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
@marcoscv-work, I'm afraid you're going to have to rebase this

**Comment by marcoscv-work** [comment]
Rebased!

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
Testing machines are on fire testing this PR!

**Comment by marcoscv-work** [comment]
ci:forward
